src/api/spec/models/user_spec.rb

```diff
@@ -870,4 +870,19 @@
       end
     end
   end
+
+  describe '#autocomplete' do
+    let(:user) { create(:confirmed_user, login: 'foobar') }
```

Comment:
For explicitly create a user you need to either call it somewhere (normally in a `before` block or in the it example) or just use the `let!`method instead that calls the creation in every single test.

BTW, I would love to see more users created with different names just to be sure that the matched ones are retrieved and not the rest. You don't only need to test that the output format is ok, you need to be sure that this finds the right objects.